eva_submission/eload_submission.py

```diff
@@ -190,8 +189,34 @@ def detect_submitted_vcf(self):
         compressed_vcf = glob.glob(os.path.join(vcf_dir, '*.vcf.gz'))
         vcf_files = uncompressed_vcf + compressed_vcf
         if len(vcf_files) < 1:
-            raise FileNotFoundError('Could not locate vcf file in in %s', vcf_dir)
-        self.eload_cfg.set('submission','vcf_files', value=vcf_files)
+            raise FileNotFoundError('Could not locate vcf file in %s', vcf_dir)
+        self.check_submitted_filenames(vcf_files)
```

Comment:
I've mentioned it earlier but `detect_submitted_vcf` is not detecting the vcf files anymore so should be renamed.
Also I've realised that the order in which `detect_submitted_vcf` and `detect_metadata_attributes` are executed matters (and the order is correct as it is) because you want to first check the concordance between metadata and filesystem then rely on the metadata only to populate the VCF files.

Reply:
I got rid of the extra function call (which wasn't really necessary) and just named the whole thing `check_submitted_filenames`.  Let me know if that's better, naming things is hard.